Command_Library/src/main/java/Depositeslides.java

```diff
@@ -0,0 +1,55 @@
+import dev.weaponboy.command_library.CommandLibrary.OpmodeEX.OpModeEX;
+import dev.weaponboy.command_library.CommandLibrary.Subsystem.SubSystem;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+
+import dev.weaponboy.command_library.CommandLibrary.Commands.LambdaCommand;
+import dev.weaponboy.command_library.CommandLibrary.Subsystem.SubSystem;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+public class Depositeslides extends SubSystem {
```

Comment:
Could you change the name to just Delivery pls. Because its also going to have the servos for the deposit arm in it as well